backend/main/strategies/staked_token_weighted_default.go

```diff
@@ -36,16 +42,28 @@ func (s *StakedTokenWeightedDefault) FetchBalance(db *s.Database, b *models.Bala
 	return b, nil
 }
 
-func (s *StakedTokenWeightedDefault) TallyVotes(votes []*models.VoteWithBalance, p *models.ProposalResults) (models.ProposalResults, error) {
+func (s *StakedTokenWeightedDefault) TallyVotes(
+	votes []*models.VoteWithBalance,
+	r *models.ProposalResults,
+) (models.ProposalResults, error) {
+
 	for _, vote := range votes {
-		p.Results[vote.Choice] += int(float64(*vote.StakingBalance) * math.Pow(10, -8))
-		p.Results_float[vote.Choice] += float64(*vote.StakingBalance) * math.Pow(10, -8)
+		if vote.StakingBalance == nil {
+			r.Results[vote.Choice] = 0.0
```

Comment:
shouldnt be setting the results to 0 because that will erase any results that have been tallied. just need `continue`